docs/analyzer/adding_recognizers.md

```diff
@@ -126,6 +126,59 @@ To add a recognizer to the list of pre-defined recognizers:
 3. Add the recognizer to the `recognizers_map` dict in the `RecognizerRegistry.load_predefined_recognizers` method. In this map, the key is the language the recognizer supports, and the value is the class itself. If your recognizer detects entities in multiple languages, add it to under the "ALL" key.
 4. Optional: Update documentation (e.g., the [supported entities list](../supported_entities.md)).
 
+### Creating ad-hoc recognizers
+
+In addition to recognizers in code, it is possible to create ad-hoc recognizers via the Presidio Analyzer API for regex and deny-list based logic.
+These recognizers, in JSON form, are added to the `/analyze` request and are only used in the context of this request.
+
+- The json structure for a regex ad-hoc recognizer is the following:
```

Comment:
nit: I would reference to the API reference instead, to avoid duplication (you can add samples there for both recognizers)

Reply:
I ended up leaving it here for simplicity, but references the OpenAPI.